frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss

```diff
@@ -51,6 +51,9 @@
   .harmful {
     justify-content: center;
   }
+  .form-control {
+    background-image: unset;
```

Comment:
what does this do?

Reply:
removes the green check icon on the review page, as it frequently disrupts the text content due to the short widths.